velox/exec/tests/RowContainerTest.cpp

```diff
@@ -47,29 +74,102 @@ class RowContainerTest : public exec::test::RowContainerTestBase {
 
     auto result = BaseVector::create(expected->type(), size, pool_.get());
     container.extractColumn(oddRows.data(), size, column, result);
-    EXPECT_EQ(size, result->size());
-    for (vector_size_t row = 0; row < size; ++row) {
-      if (row % 2 == 0) {
-        EXPECT_TRUE(result->isNullAt(row)) << "at " << row;
+    return result;
+  }
+
+  VectorPtr testExtractColumnRowNumbersForOddRows(
+      RowContainer& container,
+      const std::vector<char*>& rows,
+      int column,
+      const VectorPtr& expected) {
+    auto size = rows.size();
+
+    // Set the first row in 'oddRows' to be null, and all the
+    // even row numbers point to it.
+    std::vector<char*> oddRows(size, nullptr);
```

Comment:
oddRows name is confusing because it contains all rows, but first.

Reply:
Have changed this name.